app/views/layouts/_nav_links.html.erb

```diff
@@ -16,7 +16,7 @@
 <% end %>
 
 <%= link_to offers_path, class: "#{offers_nav_link_class(variant)}" do %>
-  Deals
+  Resources
```

Comment:
If you changed it here, did you also change the page header?

Reply:
I have changed in multiple places, let me know if that is ok.